java/arcs/android/host/parcelables/ParcelableHandleConnection.kt

```diff
@@ -28,6 +28,7 @@ data class ParcelableHandleConnection(
         parcel.writeHandle(actual.handle, flags)
         parcel.writeType(actual.type, flags)
         parcel.writeInt(actual.mode.ordinal)
+        parcel.writeString(actual.expression)
```

Comment:
See below, you could make this actual.expression?.serialize() ?: "" (see my Schema PR)